### ISSUE TITLE
Error during MFCC Feature Extration & CMVN for Training and Test set

### ISSUE THREAD

**Issue by sharanry** [mravanelli/pytorch-kaldi#81]
```
>cd ../kaldi/egs/timit/s5
>./run.sh
```

```
============================================================================
         MFCC Feature Extration & CMVN for Training and Test set          
============================================================================
steps/make_mfcc.sh --cmd queue.pl --mem 4G --nj 10 data/train exp/make_mfcc/train mfcc
utils/validate_data_dir.sh: Successfully validated data-directory data/train
steps/make_mfcc.sh: [info]: no segments file exists: assuming wav.scp indexed by utterance.
queue.pl: Error submitting jobs to queue (return status was 256)
queue log file is exp/make_mfcc/train/q/make_mfcc_train.log, command was qsub -v PATH -cwd -S /bin/bash -j y -l arch=*64* -o exp/make_mfcc/train/q/make_mfcc_train.log  -l mem_free=4G,ram_free=4G  -t 1:10 /media/sharan/Work/Coding/MachineLearning/Projects/audio-recog-for-mobile/kaldi/egs/timit/s5/exp/make_mfcc/train/q/make_mfcc_train.sh >>exp/make_mfcc/train/q/make_mfcc_train.log 2>&1
Output of qsub was: error: commlib error: got select error (Connection refused)
Unable to run job: unable to send message to qmaster using port 6444 on host "ghost": got send error
Exiting.

queue.pl: It looks like the queue master may be inaccessible.  Trying again after 20 seconts
queue.pl: Error submitting jobs to queue (return status was 256)
queue log file is exp/make_mfcc/train/q/make_mfcc_train.log, command was qsub -v PATH -cwd -S /bin/bash -j y -l arch=*64* -o exp/make_mfcc/train/q/make_mfcc_train.log  -l mem_free=4G,ram_free=4G  -t 1:10 /media/sharan/Work/Coding/MachineLearning/Projects/audio-recog-for-mobile/kaldi/egs/timit/s5/exp/make_mfcc/train/q/make_mfcc_train.sh >>exp/make_mfcc/train/q/make_mfcc_train.log 2>&1
Output of qsub was: error: commlib error: got select error (Connection refused)
Unable to run job: unable to send message to qmaster using port 6444 on host "ghost": got send error
Exiting.
error: commlib error: got select error (Connection refused)
Unable to run job: unable to send message to qmaster using port 6444 on host "ghost": got send error
Exiting.

```

**Comment by mravanelli** [comment]
I think you have to change the file timit/s5/cmd.sh to run your script
locally (see the comments in cmd.sh). If you would like to run the script
on your local machine, you have to replace queue.pl with run.pl (without
other options).

Best,

Mirco

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by sharanry** [comment]
Thank You! That worked.
